packages/realm-server/tests/helpers/indexing.ts

```diff
@@ -0,0 +1,100 @@
+import type { MatrixEvent } from 'https://cardstack.com/base/matrix-event';
+import { findRealmEvent, waitUntil } from './index';
+import { APP_BOXEL_REALM_EVENT_TYPE } from '@cardstack/runtime-common/matrix-constants';
+import { trimJsonExtension } from '@cardstack/runtime-common';
+import type {
+  IncrementalIndexEventContent,
+  IncrementalIndexInitiationContent,
+} from 'https://cardstack.com/base/matrix-event';
+import { validate as uuidValidate } from 'uuid';
+
+interface IncrementalIndexEventTestContext {
+  assert: Assert;
+  getMessagesSince: (since: number) => Promise<MatrixEvent[]>;
+  realm: string;
+}
+
```

Comment:
It looks like this is actually very flaky. If I change this matrixMessages to use some version of filter and count the incremental index events, it works "sometimes"